client/setup.py

```diff
@@ -5,85 +5,89 @@
 from setuptools import setup
 
 if sys.version_info < (3, 7, 0):
-    raise OSError(f'CLIP-as-service requires Python >=3.7, but yours is {sys.version}')
+    raise OSError(f"CLIP-as-service requires Python >=3.7, but yours is {sys.version}")
 
 try:
-    pkg_name = 'clip-client'
+    pkg_name = "clip-client"
```

Comment:
revert this kind of changes. You should enable the `pre-commit` in your local repo.